axonserver/src/main/java/io/axoniq/axonserver/localstorage/TrackingEventProcessorManager.java

```diff
@@ -177,11 +180,16 @@ public void close() {
         private final StreamObserver<InputStream> eventStream;
         private volatile CloseableIterator<SerializedEventWithToken> eventIterator;
         private volatile boolean running = true;
+        private final Set<PayloadDescription> blacklistedTypes = new CopyOnWriteArraySet<>();
+        private volatile int force = blacklistedSendAfter;
 
         private EventTracker(GetEventsRequest request, StreamObserver<InputStream> eventStream) {
```

Comment:
So, on every request from a client to receive events, the set of black listed events is shared?